tabular/src/autogluon/tabular/models/vowpalwabbit/vowpalwabbit_model.py

```diff
@@ -94,45 +76,35 @@ def _fit(self,
         final_training_data = y + X_series
         final_training_data = final_training_data.tolist()
 
+        extra_params = {
+            'cache_file': 'train.cache',
+            'holdout_off': True,
+            'quiet': True,
+        }
+
         # Initialize the model
         if self.problem_type in PROBLEM_TYPES_CLASSIFICATION:
             # Ref: https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Predicting-probabilities#multi-class---oaa
-            self.labels = list(set(y.tolist()))
-            self.oaa: int = len(self.labels)
-            print(f'Setting oaa to {self.oaa}')
-            self.model = pyvw.vw(
-                **params,
-                oaa=self.oaa,
-                cache_file='train.cache',
-                holdout_off=True,
-                probabilities=True,  # Only to be used for classification
-                quiet=True,
-            )
-        elif self.problem_type in PROBLEM_TYPES_REGRESSION:
-            self.model = pyvw.vw(
-                **params,
-                cache_file='train.cache',
-                holdout_off=True,
-                quiet=True,
-            )
+            extra_params['oaa'] = len(set(y.tolist()))
+            print(f'Setting oaa to {extra_params["oaa"]}')
```

Comment:
Switch to logging statement with level 30. By default we want all models to be silent at log 20.

```
logger.log(30, f'Setting oaa to {extra_params["oaa"]}')
```

Reply:
Addressed in killerSwitch#1